cord/client.py

```diff
@@ -192,29 +190,28 @@ def upload_video(self, file_path):
             UploadOperationNotSupportedError: If trying to upload to external
                                               datasets (e.g. S3/GPC/Azure)
         """
-        if os.path.exists(file_path):
-            short_name = os.path.basename(file_path)
-            signed_url = self._querier.basic_getter(
-                SignedVideoURL,
-                uid=short_name
-            )
-            res = upload_to_signed_url(
-                file_path,
-                signed_url,
-                self._querier,
-                Video
+        if not os.path.exists(file_path):
+            raise cord.exceptions.CordException(
+                message='{} does not point to a file.'.format(file_path)
             )
-            if res:
-                logging.info("Upload complete.")
-                logging.info("Please run client.get_dataset() to refresh.")
-                return res
-            else:
-                raise cord.exceptions.CordException(
-                    message='An error has occurred during video upload.'
-                )
+        short_name = os.path.basename(file_path)
+        signed_url = self._querier.basic_getter(
+            SignedVideoURL,
+            uid=short_name
+        )
+        res = upload_to_signed_url(
+            file_path,
+            signed_url,
+            self._querier,
+            Video
+        )
+        if res:
+            logging.info("Upload complete.")
+            logging.info("Please run client.get_dataset() to refresh.")
+            return res
         else:
             raise cord.exceptions.CordException(
-                message='{} does not point to a file.'.format(file_path)
+                message='An error has occurred during video upload.'
```

Comment:
Function `CordClientDataset.upload_video` refactored with the following changes:

- Add guard clause ([last-if-guard](https://docs.sourcery.ai/refactorings/last-if-guard/))